posthog/temporal/batch_exports/batch_exports.py

```diff
@@ -107,49 +113,76 @@ async def get_rows_count(
     return int(count)
 
 
-FIELDS = """
-DISTINCT ON (event, cityHash64(distinct_id), cityHash64(uuid))
-toString(uuid) as uuid,
-team_id,
-timestamp,
-inserted_at,
-created_at,
-event,
-properties,
--- Point in time identity fields
-toString(distinct_id) as distinct_id,
-toString(person_id) as person_id,
--- Autocapture fields
-elements_chain
-"""
+class BatchExportField(typing.TypedDict):
+    """A field to be queried from ClickHouse.
+    Attributes:
+        expression: A ClickHouse SQL expression that declares the field required.
+        alias: An alias to apply to the expression (after an 'AS' keyword).
+    """
 
-S3_FIELDS = """
-DISTINCT ON (event, cityHash64(distinct_id), cityHash64(uuid))
-toString(uuid) as uuid,
-team_id,
-timestamp,
-inserted_at,
-created_at,
-event,
-properties,
--- Point in time identity fields
-toString(distinct_id) as distinct_id,
-toString(person_id) as person_id,
-person_properties,
--- Autocapture fields
-elements_chain
-"""
+    expression: str
+    alias: str
+
+
+class BatchExportSchema(typing.TypedDict):
+    fields: list[BatchExportField]
+    values: dict[str, str]
+
+
+def default_fields() -> list[BatchExportField]:
+    """Return list of default batch export Fields."""
+    return [
+        BatchExportField(expression="toString(uuid)", alias="uuid"),
+        BatchExportField(expression="team_id", alias="team_id"),
+        BatchExportField(expression="timestamp", alias="timestamp"),
+        BatchExportField(expression="COALESCE(inserted_at, _timestamp)", alias="_inserted_at"),
```

Comment:
Since users can create their own `inserted_at` field, we rename ours to `_inserted_at`. `_inserted_at` should be included in all batch exports to allow tracking progress.